Exiled.API/Features/Log.cs

```diff
@@ -34,6 +34,22 @@ public static void Debug(object message, bool canBeSent = true)
                 Send($"[{Assembly.GetCallingAssembly().GetName().Name}] {message}", LogLevel.Debug, System.ConsoleColor.Green);
         }
 
+        /// <summary>
+        /// Sends a <see cref="LogLevel.Debug"/> level messages to the game console.
+        /// Server must have exiled_debug config enabled.
+        /// </summary>
+        /// <typeparam name="T">The inputted object's type.</typeparam>
+        /// <param name="object">The object to be logged and returned.</param>
+        /// <param name="canBeSent">Indicates whether the log can be sent or not.</param>
+        /// <returns>Returns the <typeparamref name="T"/> object inputted in <paramref name="object"/>.</returns>
+        public static T DebugObject<T>(T @object, bool canBeSent = true)
```

Comment:
This method isn't necessary, the base `Log.Debug` message takes an `object` parameter, which already allows anything to be passed.

Reply:
I am aware of that method, as I explained on the PR's description, this is eases testing a lot by allowing you to do things like `Player.List.Where(p => Log.DebugObject(p.IsDead));` without having to create an entire foreach loop that you'll then have to remove.

It is obviously not **necessary**, but a simple tool that could help with testing in some cases.

Reply:
`List.Where()`
would be the same as
`
foreach (Player player in Player.List)
    if (player.IsDead)`
but with an extra GetEnumerator for seemingly no reason.

Reply:
What I meant wasn't that this is supposed to be something to solve an issue currently present, it's mainly just a small niche feature that could make debugging things a little simpler, there's many cases where this could be applied.